values.yaml

```diff
@@ -78,12 +78,16 @@ podSecurityContext:
 
 securityContext:
   runAsNonRoot: true
-  runAsUser: 33  # www-data
+  runAsUser: 33 # www-data
   readOnlyRootFilesystem: true
   # capabilities:
   #   drop:
   #   - ALL
 
+containerSecurityContext:
+  runAsNonRoot: true
+  runAsUser: 33 # www-data
+
```

Comment:
So now we must specify the `containerSecurityContext` also when it is the same as the `securityContext` isn't it? Why don't we inherit from the values of `common` and set it there as default to be the same as `securityContext`? From my understanding it's the most common use case, no?

Reply:
No, it's not in the `containerSecurityContext` we didn't have `readOnlyRootFilesystem` with not allowed...